### ISSUE TITLE
Add legal_awareness dataset for leisure sea fishing in France to live deploy list

### ISSUE THREAD

**Issue by AlasdairGray** [BioSchemas/specifications#409]
I have used bioschemas.org/Taxon to qualify some legal_awareness dataset for leisure sea fishing in France, fishes, shells ... This is in test since about a year now for all species that are targeted by some legislation.

https://search.google.com/structured-data/testing-tool#url=https%3A%2F%2Fwww.opalesurfcasting.net%2Fconnaissances%2Ffaune-et-flore%2Fle-saumon-de-l-atlantique-salmo-salar.html

https://search.google.com/test/rich-results?id=EnYla7r4hYxRyh_uyf6XgA

_Originally posted by @eldk in https://github.com/BioSchemas/specifications/issues/321#issuecomment-570834625_

**Comment by AlasdairGray** [comment]
@eldk Thanks for adding this to our list of live deploys. I'm sorry I hadn't managed to do this for you.

The markup looks to be mostly correct. The one thing is that the [Taxon Profile](https://bioschemas.org/profiles/Taxon/) requires a link to a taxon rank. 

If you are having difficulties knowing what to do with this, @frmichel would be the person to ask.

**Comment by eldk** [comment]
Thanks @AlasdairGray,

I have added taxonRank as text for the moment, using 'species'. I will look at use TDWG TaxonRank ontology in next days.

**Comment by frmichel** [comment]
Just for information, some TDWG ontologies are deprecated today, although new work is on track to come up with an updated version thereof.

However the idea of taxonRank is to give users flexibility wrt. to what they use to denote the ranks: URIs, or text. 
In an ideal world where semantic web practices would be adopted everywhere, we would only need URIs. And we have many of them: if you lookup the [species rank](http://taxref.mnhn.fr/lod/taxrank/Species) in TAXREF-LD, you'll see I have aligned it with TDWG, NCBI, Geospecies, TaxonConcept and the Taxonomic rank vocabulary.

But this is no ideal world, and many people still use strings for ranks. Typically because lots of data come from Darwin Core archives that are basically XML files where the rank is a string. 

So this is the reason why I proposed to have the two options, text or URI.
